nextjs/packages/next/server/config-shared.ts

```diff
@@ -15,12 +15,12 @@ export type LogLevel = 'trace' | 'debug' | 'info' | 'warn' | 'error' | 'fatal'
 export type LogType = 'json' | 'pretty' | 'hidden'
 
 export function loadConfigAtRuntime() {
-  if (!process.env.BLITZ_APP_DIR) {
+  if (!process.cwd()) {
     throw new Error(
-      'Internal Blitz Error: process.env.BLITZ_APP_DIR is not set'
+      'Internal Blitz Error: process.cwd() is not set'
     )
   }
-  return loadConfigProduction(process.env.BLITZ_APP_DIR)
+  return loadConfigProduction(process.cwd())
 }
```

Comment:
There are some cases when `BLITZ_APP_DIR` is necessary, so I think the safe approach here would be to do something like this instead:

```js
export function loadConfigAtRuntime() {
  // possibly a log that `BLITZ_APP_DIR` is not set, using `process.cwd()` instead
  return loadConfigProduction(process.env.BLITZ_APP_DIR || process.cwd())
}
```

Reply:
good suggestion, addressed in f183f8f